packages/clay-data-provider/src/__tests__/LRUCache.ts

```diff
@@ -0,0 +1,63 @@
+/**
+ * © 2019 Liferay, Inc. <https://liferay.com>
+ *
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+import {LRUCache} from '../LRUCache';
+
+let cache: LRUCache<string, {}>;
+
+describe('LRUCache', () => {
+	beforeEach(() => {
+		cache = new LRUCache(3);
+	});
+
+	afterEach(() => {
+		cache.reset();
+	});
+
+	it('throw error if initialized with maxSize 0', () => {
```

Comment:
"throw error if" isn't grammatical, so please change to: "throws if" or "throws an error if".

Same in the next example.